io.github.Predidit.Kazumi.yaml

```diff
@@ -0,0 +1,100 @@
+id: io.github.Predidit.Kazumi
+runtime: org.gnome.Platform
```

Comment:
do you really need the GNOME platform?

Reply:
I believe it's the best way to get webkit-gtk4.1.

Reply:
Yes